clients/node/client-rds-data-node/runtimeConfig.browser.ts

```diff
@@ -0,0 +1,30 @@
+import { invalidFunction } from '@aws-sdk/invalid-dependency';
```

Comment:
Prettier wasn't run on this file.
Please cut a separate issue for tracking if git hooks aren't working

Reply:
How do you know it?

Reply:
Single quotes 😉

If this is an issue with git-hooks, it can be tracked in a separate issue

Reply:
It turns out my local workspace's formatter did set to vscode default ones. So I added a vscode.settings: `"editor.defaultFormatter": "esbenp.prettier-vscode"`